SMLHelper/FMod/FModMultiSounds.cs

```diff
@@ -79,13 +81,14 @@ public FModMultiSounds(string[] soundPaths, MODE mode, string busPath, bool rand
             this.randomizeSounds = randomizeSounds;
         }
 
+        [Obsolete("Deprecated. Use TryPlaySound instead.")]
         Channel IFModSound.PlaySound()
         {
             if (_sounds is {Length: > 0})
             {
                 if (randomizeSounds)
                 {
-                    return AudioUtils.PlaySound(_sounds[Random.Range(0, _sounds.Length)], _bus);
+                    return AudioUtils.PlaySound(_sounds[Random.Range(0, _sounds.Length - 1)], _bus);
```

Comment:
`Random.Range` for integers is min-inclusive & max-exclusive as opposed to range for floats, so it should be 0 to length, or the last sound in the collection wont be ever played.
```suggestion
                    return AudioUtils.PlaySound(_sounds[Random.Range(0, _sounds.Length)], _bus);
```